### ISSUE TITLE
History ValueError with xonfig wizard on fresh install

### ISSUE THREAD

**Issue by gforsyth** [scopatz/xonsh#627]
On the very first run of `xonsh` after installation on a machine that has never seen it:

```Bash
 ~  xonsh                  
Could not find xonsh configuration or run control files.
Would you like to run the xonsh configuration wizard now, yes or no [default: no]? yes
ValueError: History index cannot be negative
```

Then `$PROMPT` is replaced by `Would you like to run the xonsh configuration wizard now, yes or no [default: no]? `

If you type `exit` and then relaunch, there is no issue since there is then a history of at least the `exit` command, but all the same, this will probably tick off new users.

**Comment by scopatz** [comment]
Thanks for reporting @gforsyth. fix incoming, hopefully.

**Comment by scopatz** [comment]
Do you happen to have the full traceback?

**Comment by gforsyth** [comment]
Hrrr... here's *a* traceback.  If I set `$XONSH_SHOW_TRACEBACK=True` in my `xonshrc` then I have a `xonshrc` and the wizard doesn't start up automagically on startup.  But I think I triggered a similar error by trying to accept default values without a history file.

```
Add a foreign shell, yes or no [default: no]?   File "/home/gil/.local/lib/python3.4/site-packages/xonsh/proc.py", line 338, in wrapped_simple_command
    r = f(args, i)
  File "/home/gil/.local/lib/python3.4/site-packages/xonsh/aliases.py", line 168, in xonfig
    return main(args)
  File "/home/gil/.local/lib/python3.4/site-packages/xonsh/xonfig.py", line 285, in main
    return _MAIN_ACTIONS[ns.action](ns)
  File "/home/gil/.local/lib/python3.4/site-packages/xonsh/xonfig.py", line 207, in _wizard
    pv.visit()
  File "/home/gil/.local/lib/python3.4/site-packages/xonsh/wizard.py", line 469, in visit
    rtn = super().visit(node)
  File "/home/gil/.local/lib/python3.4/site-packages/xonsh/wizard.py", line 292, in visit
    rtn = meth(node)
  File "/home/gil/.local/lib/python3.4/site-packages/xonsh/wizard.py", line 520, in visit_wizard
    self.visit(child)
  File "/home/gil/.local/lib/python3.4/site-packages/xonsh/wizard.py", line 469, in visit
    rtn = super().visit(node)
  File "/home/gil/.local/lib/python3.4/site-packages/xonsh/wizard.py", line 292, in visit
    rtn = meth(node)
  File "/home/gil/.local/lib/python3.4/site-packages/xonsh/wizard.py", line 565, in visit_while
    while node.cond(visitor=self, node=node):
  File "/home/gil/.local/lib/python3.4/site-packages/xonsh/wizard.py", line 264, in truefalse_cond
    rtn = visitor.visit(tf)
  File "/home/gil/.local/lib/python3.4/site-packages/xonsh/wizard.py", line 469, in visit
    rtn = super().visit(node)
  File "/home/gil/.local/lib/python3.4/site-packages/xonsh/wizard.py", line 292, in visit
    rtn = meth(node)
  File "/home/gil/.local/lib/python3.4/site-packages/xonsh/wizard.py", line 540, in visit_input
    x = self.shell.singleline(**self.shell_kwargs)
  File "/home/gil/.local/lib/python3.4/site-packages/xonsh/readline_shell.py", line 117, in singleline
    readline.remove_history_item(pos)
ValueError: History index cannot be negative
```

**Comment by scopatz** [comment]
Thanks!

**Comment by gforsyth** [comment]
Resolved by #628
